fix(tutorials): added tags corrected typos and fixed indentation MTA-1585 (#225)
scaleway/docs-content: tutorials/nginx-reverse-proxy/index.mdx

A [Nginx HTTPS reverse proxy](https://nginx.org) is an intermediary proxy service which takes a client request, passes it on to one or more servers, and subsequently delivers the server’s response back to the client. While most common applications are able to run as web server on their own, the Nginx web server is able to provide a number of advanced features such as load balancing, TLS/SSL capabilities and acceleration that most specialized applications lack. By using a Nginx reverse proxy all applications can benefit from these features.

> There are important benefits of setting up a Nginx HTTPS reverse proxy:

Reviewer: ```suggestion
There are significant benefits to setting up a Nginx HTTPS reverse proxy:
```
